scrapers_next/vt/committees.py

```diff
@@ -0,0 +1,76 @@
+from lxml.html import fromstring
+from lxml.html.clean import Cleaner
+from lxml.etree import ParserError
+from spatula import JsonListPage, URL, SkipItem
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeList(JsonListPage):
+    source = URL(
+        "https://legislature.vermont.gov/committee/loadList/2024/",
+        headers={
```

Comment:
Do we need this header? If we don't have to lie about our user agent, I'd rather we didn't.

Reply:
okk